recipe/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
     - llvm-openmp  # [osx]
     - libgomp      # [linux]
   host:
-    - numpy >=1.14.5
+    - numpy
```

Comment:
I'm not sure I understand why `numpy >=1.14.5` is interfering with pinning to a more recent version, but if that's the case and this is the fix then sure...

Reply:
I believe the issue is that the pin in the `meta.yaml` overrides that in the [global conda-forge pinnings](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml). The meta file says anything greater than 1.14.5 so builds with the latest version available at the time of build so forces anything to use that version but we want to pin back to match the conda-forge numpy pinnings so there is a conflict.